diff --git a/lib/index.js b/lib/index.js
index 1167aa26f6ad3cbacde07230386a54270e0bcdb9..0433a0c3b4d63fb2d3caf7b5165f9e4028b23383 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -179,12 +179,6 @@ function createLoadableComponent(loadFn, options) {
     _proto._loadModule = function _loadModule() {
       var _this2 = this;
 
-      if (this.context && Array.isArray(opts.modules)) {
-        opts.modules.forEach(function (moduleName) {
-          _this2.context.report(moduleName);
-        });
-      }
-
       if (!res.loading) {
         return;
       }
